src/smsfusion/_ahrs.py

```diff
@@ -201,7 +201,7 @@ def euler(self, degrees: bool = True) -> NDArray[np.float64]:
         Returns
         -------
         euler : numpy.ndarray
-            Euler angles, specifically:  alpha (roll), beta (pitch) and gamma (yaw)
+            Euler angle vector, specifically: alpha (roll), beta (pitch) and gamma (yaw)
```

Comment:
Technically, Euler angles are not  vectors. Let's not use that term. May be "sequence" instead?

Reply:
Same for all the occurenes elsewhere.

Reply:
Good point. I can fix that.